test/e2e/apimachinery/namespace.go

```diff
@@ -246,4 +248,29 @@ var _ = SIGDescribe("Namespaces [Serial]", func() {
 	ginkgo.It("should always delete fast (ALL of 100 namespaces in 150 seconds) [Feature:ComprehensiveNamespaceDraining]",
 		func() { extinguish(f, 100, 0, 150) })
 
+	ginkgo.It("should patch a Namespace", func() {
+		ginkgo.By("creating a Namespace")
+		namespaceName := "nspatchtest"
+		_, err := f.ClientSet.CoreV1().Namespaces().Create(&v1.Namespace{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: namespaceName,
+			},
+		})
+		framework.ExpectNoError(err, "failed creating Namespace")
+
+		ginkgo.By("patching the Namespace")
+		nspatch := `{"metadata":{"labels":{"testLabel":"testValue"}}}`
+		_, err = f.ClientSet.CoreV1().Namespaces().Patch(namespaceName, types.StrategicMergePatchType, []byte(nspatch))
+		framework.ExpectNoError(err, "failed to patch Namespace")
```

Comment:
If it failed here, wouldn't we orphan the new ns we made? I'm not exactly certain though how the `Expect...` works w.r.t. defers; even if you did a 'defer deleteNS(...)` would the namespace still be orphaned on failure?

Reply:
@johnSchnake 
cleanup of failure (defer) is not a common pattern in this file and other tests.
the namespace will be orphaned, but essentially this is just a test, so it's fine.

@BobyMCbobs 
`defer deleteNamespace(f, namespaceName)` will now do a second delete that will always fail on success, if the previous code passes:
```
		ginkgo.By("deleting the Namespace")
		err = f.ClientSet.CoreV1().Namespaces().Delete(namespaceName, &metav1.DeleteOptions{})
		framework.ExpectNoError(err, "failed to delete the test Namespace")
```

i don't think we have to have the deferred delete.

Reply:
OK, I'll remove it.